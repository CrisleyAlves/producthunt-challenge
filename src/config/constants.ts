export const POST_ORDER_TYPE = {
  POPULAR: {
    label: "Popular",
    value: "VOTES",
  },
  NEWEST: {
    label: "Newest",
    value: "NEWEST",
  },
};

export const NUMBER_OF_POSTS = 20;

export const PROFILE_PICTURE =
  "https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y";
