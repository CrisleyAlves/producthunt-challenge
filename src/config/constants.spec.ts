import { describe, it, expect } from "vitest";
import * as constants from "./constants";

describe("constants", () => {
  it("should match the constants snapshot", () => {
    expect(constants).toMatchSnapshot();
  });
});
