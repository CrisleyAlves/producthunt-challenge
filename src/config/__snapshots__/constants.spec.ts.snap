// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`constants > should match the constants snapshot 1`] = `
{
  "NUMBER_OF_POSTS": 20,
  "POST_ORDER_TYPE": {
    "NEWEST": {
      "label": "Newest",
      "value": "NEWEST",
    },
    "POPULAR": {
      "label": "Popular",
      "value": "VOTES",
    },
  },
  "PROFILE_PICTURE": "https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y",
}
`;
